README.md

```diff
@@ -126,375 +126,122 @@ If your car is not listed above, then there has been no documented information o
+`commaai/devel` is the same as the official v0.9.8 release, except with TSK support enabled.
 
-C3X will display a splash screen with Comma logo.
+![](img/v3.reboot.jpg)
 
-4B-2-5. Run the key extraction script.
-```sh
-./extract_keys.py
-```
+<details><summary>Troubleshooting</summary>
 
-> [!Tip]
-> If you restarted C3X, be sure to run `pkill -f openpilot` on each C3X restart before running `./extract_keys.py`.
+1. `commaai/devel` is the only official release branch from comma.ai that supports TSK vehicles. If you install a branch without TSK support, openpilot won't be able to drive your car. See [Forks](#forks) for more information.
+</details>
 
-If you see something like this, the key extraction was successful.
-```sh
-comma@comma-71b93b83:/data/openpilot/secoc$ ./extract_keys.py
-INFO: connecting to panda 2c0004004450383632311333
-Getting application versions...
- - APPLICATION_SOFTWARE_IDENTIFICATION (application) b'\x018965B4509100\x00\x00\x00\x00'
- - APPLICATION_SOFTWARE_IDENTIFICATION (bootloader)  b'\x01!!!!!!!!!!!!!!!!'
-
-Security Access...
- - SEED: 36552fe27172c99222eec3a9b9bd1f28
- - KEY: b7b55ba16369bba912b7aa4c06e6c35e
- - Key OK!
-
-Preparing to upload payload...
- - Write data by identifier 0x201 00000000000000000000000000000000
- - Write data by identifier 0x202 00000000000000000000000000000000
-
-Upload payload...
- - Request download
- - Transfer data 0
- - Transfer data 1
- - Transfer data 2
- - Transfer data 3
-
-Verify payload...
- - Routine control 0x10f0 OK!
-
-Trigger payload...
-
-Dumping keys...
-100%|████████████████████████| 448/448 [00:00<00:00, 14293.36it/s]
-
-ECU_MASTER_KEY    82667ef509b9f07a134aaf89d4973c68
-SecOC Key (KEY_4) 0123456789abcdef0123456789abcdef
-
-SecOC key written to param successfully!
-comma@comma-71b93b83:/data/openpilot/secoc$
-```
-
-This 32 digit hexadecimal number is your key.
-> SecOC Key (KEY_4) **0123456789abcdef0123456789abcdef**
-
-Archive and don't lose the key so that you don't need to extract it again. Perhaps email it to yourself.
-
-> [!WARNING]
-> It's theoretically possible for someone to remotely hack your car with the key under very specific circumstances. You don't need to protect the key like it's your bank password, but still don't post it on Discord.
-
-#### Step 4B-3. Debugging
-
-4B-3-1. If you see any of these error messages
+### Step 6. Calibrate & Clean up
 
-* `panda.python.uds.MessageTimeoutError: timeout waiting for response`
-* `panda.python.uds.InvalidServiceIdError: invalid response service id: 0x50` or similar
-* `Can't read application software identification. Please cycle ignition.`
+C3X should show to the 15mph calibration screen.
 
-Turn off the car, put it back into `Not Ready to Drive` mode, and then try again.
+![](img/v3.calibrate.jpg)
 
-Be sure to kill openpilot process if you restarted C3X.
+If you're able to calibrate and then use openpilot to use the steering wheel (aka "lat support"), you can clean up the cables and put the covers back on.
```

Comment:
Not: wording... maybe something like "If you're able to calibrate and use openpilot to control the steering wheel ..."

Reply:
Changed